test/integration/authorization_test.go

```diff
@@ -302,14 +302,15 @@ func TestAuthorizationResolution(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
-	// try to add Valerie to a non-existent role, looping until it is true due to
-	// the policy cache taking time to react
+	// try to add Valerie to a non-existent role, looping until it is done.
+	// If a Not Found error is raised, simply retry as this is likely due to
+	// the role cache taking time to react
```

Comment:
I think in general this wait no longer makes any sense (there is no role cache).

```go
if err := addValerie.AddRole(); err != nil {
    t.Fatalf("unexpected error: %v", err)
}
```

@deads2k any thoughts?